packages/react/src/experimental/Navigation/Header/ProductUpdates/index.tsx

```diff
@@ -94,7 +94,9 @@ const ProductUpdates = ({
         if (open && updates === null) {
           invokeGetUpdates()
         }
-        onOpenChange(open)
+        if (open) {
+          onOpenChange(open)
+        }
```

Comment:
`onOpenChange` should trigger on any change state - opening and closing. That's why we pass a boolean to convey what is the change open→closed or closed→open